### DISCUSSION TITLE
attu is connecting to milvus server

### DISCUSSION THREAD

**Discussion by vihariazure** [milvus-io/milvus#38387] [Q&A and General discussion]
hi team below is the reference link iam going 

https://milvus.io/docs/v2.2.x/attu_install-docker.md

both attu and milvus is running in local system

**1) first i tired with docker**
docker run -p 8000:3000  -e MILVUS_URL={your machine IP}:19530 zilliz/attu:v2.2.8

i replaced with my ip address which taken form the network&internet >> wifi

ipv4 address i took 

then i replaced in the above code . it ran success fully

i the document it mentioned use http://{ your machine IP }:8000
but it is not reachable i used local host . attu ui came 
i given milvus url as {your machine IP}:19530

**i got connection error (" Error: 14 UNAVAILABLE: No connection established")**
**2) i tired with docker compose it works** 
i tired to connect with http://{ your machine IP }:8000 by replacing my ip address it unable to connect
then i used local host it connected . in the milvus url it automatically took (milvus-standalone:19530). which is mentioned under env of attu of docker compose yml file.

it connected it shows everthing 

now in the vm milvus is running 
i need to connect to that milvus form the attu 

so i changed the MILVUS_URL: milvus-standalone:19530 to ip_address_of_vim:19530
**it shows connectivity issue again (Error: 14 UNAVAILABLE: No connection established )**

**Comment by yhmo** [comment]
Maybe the Attu v2.2.8 isn't compatible with milvus v2.4/v2.5. 
Wait @shanghaikid to comment.

**Comment by shanghaikid** [comment]
I think you should use 2.4.x attu, but you may face some rendering issue if you are using newer features。  
Attu 2.5.0 will be released soon, stay tuned.
